monticore-grammar/src/main/java/de/monticore/types3/generics/util/TypeParameterRelationsDefaultDelegatee.java

```diff
@@ -0,0 +1,93 @@
+package de.monticore.types3.generics.util;
+
+import de.monticore.symbols.basicsymbols._symboltable.IBasicSymbolsScope;
+import de.monticore.types.check.SymTypeExpression;
+import de.monticore.types.check.SymTypeInferenceVariable;
+import de.monticore.types.check.SymTypeVariable;
+import de.monticore.types3.generics.TypeParameterRelations;
+import de.monticore.types3.util.SymTypeCollectionVisitor;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Default delegatee for TypeParameterRelations.
+ * This itself only delegates to the individual implementations.
```

Comment:
```suggestion
 * This itself delegates to the individual implementations.
```

(oder delegiert `_getIncludedInferenceVariables`? Bin mir da selber nicht so sicher 😅 )